gcp-compute-service-adapter.js

```diff
@@ -164,18 +191,22 @@ async function rollback(createdResources, action, settings) {
 async function createGCPServices(loadBalancerResourcesData, action, settings) {
   const results = {};
   const createdResources = [];
+  const credentials = helpers.getCredentials(action.params, settings);
+  const project = helpers.getProject(action.params, settings);
```

Comment:
Those two lines are called for every single function in this file, I'd suggest extracting / injecting those values somehow.

Reply:
I planned to extract that. Fixed